action.yaml

```diff
@@ -372,3 +387,12 @@ runs:
         DepsBranch
         DepsDev
       dependencies: ${{ inputs.dependencies }}
+
+  - name: Restore DESCRIPTION
+    if: ${{ inputs.restore-description == 'true' }}
```

Comment:
Shall we introduce identical if statement in the copy step earlier?

Reply:
Okay.